packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -20,6 +20,10 @@
     }
 }
 
+.landscape-blocker {
```

Comment:
Right now, when we load dtrader and navigate to different apps such as trader's hub the landscape blocker is not shown. I think it's because once this CSS is loaded it stays in the context.

Reply:
Can we instead move this logic to the useEffect hook in DTrader?

Reply:
In this case i will move it core to check the current route, but we will remove it later anyway right? Should it be handled as well?